bus/bus.go

```diff
@@ -396,11 +402,8 @@ func (b *Bus) Handler() http.Handler {
 		"POST   /alerts/dismiss":  b.handlePOSTAlertsDismiss,
 		"POST   /alerts/register": b.handlePOSTAlertsRegister,
 
-		"GET    /autopilots":    b.autopilotsListHandlerGET,
-		"GET    /autopilot/:id": b.autopilotsHandlerGET,
-		"PUT    /autopilot/:id": b.autopilotsHandlerPUT,
-
-		"PUT    /autopilot/:id/host/:hostkey/check": b.autopilotHostCheckHandlerPUT,
+		"GET    /autopilot": b.autopilotHandlerGET,
```

Comment:
@ChrisSchinnerl `/autopilot` or `/autopilot/config`? I'm asking because in the UI it's the latter. Before pinging Alex to update I'm wondering what makes the most sense. I think I like the shortest version, so the former.